framework/decode/file_processor.cpp

```diff
@@ -628,6 +628,77 @@ bool FileProcessor::ProcessMetaData(const format::BlockHeader& block_header, for
             HandleBlockReadError(kErrorReadingBlockHeader, "Failed to read display message meta-data block header");
         }
     }
+    else if (meta_data_type == format::MetaDataType::kCreateHardwareBufferCommand_deprecated)
+    {
```

Comment:
Could it add a wrong message for not using this type, the user should capture it again?

Reply:
Yes, that's a good idea and I'll see about adding that!

Reply:
Added directly to `dev` as 9e2f5ecd